src/rust/engine/process_execution/bazel_protos/src/conversions.rs

```diff
@@ -17,6 +26,52 @@ impl<'a> From<&'a super::remote_execution::Digest> for Result<hashing::Digest, S
   }
 }
 
+impl From<crate::google::longrunning::Operation> for crate::operations::Operation {
+  fn from(op: crate::google::longrunning::Operation) -> Self {
+    let mut dst = Self::new();
+    dst.set_name(op.name);
+    dst.set_metadata(prost_any_to_gcprio_any(op.metadata.unwrap()));
+    dst.set_done(op.done);
+    match op.result {
+      Some(crate::google::longrunning::operation::Result::Response(response)) => {
+        dst.set_response(prost_any_to_gcprio_any(response))
+      }
+      Some(crate::google::longrunning::operation::Result::Error(status)) => {
+        dst.set_error(prost_status_to_gcprio_status(status))
+      }
+      None => {}
+    };
+    dst
+  }
+}
+
+pub fn prost_any_to_gcprio_any(any: prost_types::Any) -> protobuf::well_known_types::Any {
+  let prost_types::Any { type_url, value } = any;
+  let mut dst = protobuf::well_known_types::Any::new();
+  dst.set_type_url(type_url);
+  dst.set_value(value);
+  dst
+}
+
+pub fn prost_status_to_gcprio_status(status: crate::google::rpc::Status) -> crate::status::Status {
```

Comment:
I guess that because this is public outside of the crate, it doesn't need an `allow_unused`?